pkgs/applications/video/catt/default.nix

```diff
@@ -19,6 +19,11 @@ buildPythonApplication rec {
     youtube-dl
   ];
 
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace 'Click>=7.1.2,<8' 'click' \
```

Comment:
`click>8` contains breaking changes. It would probably be better to override `click`.

Reply:
https://github.com/skorokithakis/catt/pull/311
it was pinned because it didn't work with python 3.5

Reply:
opened an issue  #354